src/apps/deskflow-gui/Hotkey.h

```diff
@@ -56,20 +57,23 @@ class Hotkey
 protected:
   KeySequence &keySequence()
   {
-    return m_KeySequence;
+    return m_keySequence;
   }
   void setKeySequence(const KeySequence &seq)
   {
-    m_KeySequence = seq;
+    m_keySequence = seq;
   }
   ActionList &actions()
   {
-    return m_Actions;
+    return m_actions;
   }
 
 private:
-  KeySequence m_KeySequence;
-  ActionList m_Actions;
+  KeySequence m_keySequence;
+  ActionList m_actions;
+  inline static const QString kSectionActions = QStringLiteral("actions");
+  inline static const QString kMousebutton = QStringLiteral("mousebutton(%1)");
```

Comment:
Very minor nit, happy to ignore

```suggestion
  inline static const QString kMouseButton = QStringLiteral("mousebutton(%1)");
```

Reply:
Ignoring nit